src/firefly_exchange_client/sockets.py

```diff
@@ -80,13 +80,13 @@ async def subscribe_global_updates_by_symbol(self,symbol: MARKET_SYMBOLS):
             if not self.connection_established:
                 raise Exception("Socket connection is established, invoke socket.open()")
 
-            sio.emit('SUBSCRIBE',[
+            resp = sio.call('SUBSCRIBE',[
             {
                 "e": SOCKET_EVENTS.GLOBAL_UPDATES_ROOM.value,
                 "p": symbol.value,
             },
             ])
-            return True
+            return resp["success"]
```

Comment:
Can we return the whole response here?
We send message as well from the server. So in case of failure , user will able to know the reason as well that why their subscription is failed.

> const data: userSubscriptionAck = {
            success: false,
            message: "Subscription failed due to incorrect event name"
        };

Reply:
Yes.